fastapi/exceptions.py

```diff
@@ -153,6 +153,21 @@ def __init__(self, errors: Sequence[Any]) -> None:
     def errors(self) -> Sequence[Any]:
         return self._errors
 
+    def __str__(self) -> str:
+        message = f"{len(self._errors)} validation error"
+        if len(self._errors) != 1:
+            message += "s"
```

Comment:
smokeshow coverage is red because of this line. =/

Reply:
It's harder to review because it lacks coverage on this line that pluralizes a string? With all due respect, give me a break. If this one is too hard, please consider reviewing https://github.com/fastapi/fastapi/pull/12970.

Reply:
I find your comment pretty disrespectful.

I have quite some years of experience in open-source maintenance, including writing, reviewing and merging PRs. Countless times I've seen PRs being submitted that were "trivial" with comments such as just "please just merge". But as soon as tests get properly added (incl. tests that ensure the coverage stays 100%), suddenly little bugs pop up. That's fine, that's a normal dev process, but it's the main reason why we generally ask for adequate tests ànd a passing CI before the team starts reviewing.

I want to iterate that we really appreciate user contributions and PRs. Unfortunately there's a huge load of open PRs (280) and we do our best to manage this queue. If users can help by (also) adding adequate tests, keeping the branch up-to-date wist `master` and ensuring the CI passes, then yes that will certainly help us in the reviewing process.

Reply:
I will put this PR in draft as long as the CI is failing.

Reply:
> I find your comment pretty disrespectful.

I hope you can understand that the disrespect was not the intent. I won't make excuses, you can see for yourself [how long I've been waiting for reviews](https://github.com/fastapi/fastapi/pulls/tamird).

> I have quite some years of experience in open-source maintenance, including writing, reviewing and merging PRs. Countless times I've seen PRs being submitted that were "trivial" with comments such as just "please just merge". But as soon as tests get properly added (incl. tests that ensure the coverage stays 100%), suddenly little bugs pop up. That's fine, that's a normal dev process, but it's the main reason why we generally ask for adequate tests ànd a passing CI before the team starts reviewing.

I too have quite some years of experience in open-source. I understand, of course, that you want to keep things healthy and coverage is part of that. But requests of this kind inherently require trust between the author and the reviewer; you ask that I invest more time _in advance_ of you investing _any_ time. Coupled with the prior experience of waiting for reviews for the past 7 months, this trust has not been earned. [Other contributors](https://github.com/fastapi/fastapi/pulls?q=is%3Apr+is%3Aopen+sort%3Acomments-asc) have not fared much better.

> I want to iterate that we really appreciate user contributions and PRs. Unfortunately there's a huge load of open PRs (280) and we do our best to manage this queue. If users can help by (also) adding adequate tests, keeping the branch up-to-date wist `master` and ensuring the CI passes, then yes that will certainly help us in the reviewing process.

Sure. I have now addressed your request. Thank you.